.claude/skills/run-ci-tests/bin/report.py

```
@@ -0,0 +1,199 @@
+    args = sys.argv[1:]
+    if not args: sys.exit(__doc__)
+    log_dir = args[0]; out = None; title = "Local CI test run"
+    i = 1
+    while i < len(args):
+        if args[i] == "--out": out = args[i + 1]; i += 2
+        elif args[i] == "--title": title = args[i + 1]; i += 2
+        else: sys.exit(f"unknown option {args[i]}")
+    summary = read_summary(log_dir)
+    logs = [p for p in glob.glob(os.path.join(log_dir, "*.log")) if not os.path.basename(p).startswith("chain-")]
+    infos = [parse_log(p) for p in logs]
+    rank = lambda i: (ORDER.index(i["name"]) if i["name"] in ORDER else len(ORDER), i["name"])
```

Ruff (0.16.2)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `rank` as a `def`

(E731)